### DISCUSSION TITLE
Does Anyone Faced This Unexpected Problem While Doing "mvn clean install"

### DISCUSSION THREAD

**Discussion by piyush-1234** [checkstyle/checkstyle#14218] [Q&A] [answered]
i unexpectedly got this problem in my local, when i tried to build master after updating it from upstream/master, 
looks like this is my machine problem, related to jdk maybe, 
i tried with git reset and all although, but this is the thing i am getting in my terminal, 
i dont know why that test case is failing, getting FileSystemException

![image](https://github.com/checkstyle/checkstyle/assets/76208964/7954352e-b902-4a6e-a9a1-b93ad068c103)

**Comment by rnveach** [comment, marked as answer]
`testSymbolicLinkResolution` was just recently added (yesterday) at https://github.com/checkstyle/checkstyle/commit/2e99389ba88a7b802383eb79c48b611237ec7de9#diff-6beb025a797ba5f4151d47147b877d7f58bbeaf28e78f235604aaf5f5e9cd415R304 .

I assume there is some issue that our CI did not catch when we added the test.

**Reply by nrmancuso** [reply]
Yep, @piyush-1234 thanks for bringing this to our attention. Can you try this and report back?
```bash
git checkout 86b9a28f763e0e5b5859b581a276bc290078e84a
mvn clean verify
````

this is the last commit before the one that @rnveach mentioned above, if everything is ok on this one then we have confirmed that the issue lies in `2e99389` as suspected.

**Reply by piyush-1234** [reply]
gotcha, will it be a revertable change, or do we have a counter resolution. 

is your local building?

**Reply by nrmancuso** [reply]
I use Fedora, I cannot reproduce this. Please do:
1. Create an issue
2. Show cli output of command and failure message (not a screenshot; you can omit stuff in between that isn't important)
3. Share your windows version, java version, and any other relevant information

**Reply by piyush-1234** [reply]
sure @nrmancuso , on it

**Reply by nrmancuso** [reply]
@piyush-1234 thanks for the help with diagnostics, I have changed this discussion to Q&A and marked this answer as selected based on your comment at https://github.com/checkstyle/checkstyle/discussions/14218#discussioncomment-7999347

**Reply by piyush-1234** [reply]
@nrmancuso , #14219 is the issue got created.

**Comment by piyush-1234** [comment]
> Yep, @piyush-1234 thanks for bringing this to our attention. Can you try this and report back?
> 
> ```shell
> git checkout 86b9a28f763e0e5b5859b581a276bc290078e84a
> mvn clean verify
> ```
> 
> this is the last commit before the one that @rnveach mentioned above, if everything is ok on this one then we have confirmed that the issue lies in `2e99389` as suspected.

sure @nrmancuso ,

**Comment by piyush-1234** [comment]
> Yep, @piyush-1234 thanks for bringing this to our attention. Can you try this and report back?
> 
> ```shell
> git checkout 86b9a28f763e0e5b5859b581a276bc290078e84a
> mvn clean verify
> ```
> 
> this is the last commit before the one that @rnveach mentioned above, if everything is ok on this one then we have confirmed that the issue lies in `2e99389` as suspected.

@nrmancuso this **86b9a28f763e0e5b5859b581a276bc290078e84a** looks fine, got built success, 

![image](https://github.com/checkstyle/checkstyle/assets/76208964/bd5a0318-bf5d-4f82-8e38-05549dcf3362)
